deploy.py

```diff
@@ -473,6 +473,8 @@ def _check_invariants(self):
     print("Testing invariants in customization.rs...")
     self.checked_command_output(
         ["cargo", "test", "--features=std", "--lib", "customization"])
+    self.checked_command_output(
+        ["cargo", "test", "--features=std", "--lib", "api::customization"])
```

Comment:
I'm surprised we need this. The command above should also contain `api::customization` because it just greps anything in the test name. I would remove it if it's the case because we test twice.

Reply:
Ah I didn't read carefully. My intention was: Make sure that both tests are run before deploying. If no change is required for that, good with me.

Reply:
Fixed, need approve again.